README.md

```diff
@@ -119,6 +119,39 @@ All the tests are designed to test end-to-end scenarios, and there are some requ
 
 -   MQTT v5 broker (like mosquitto v1.6.7 server running)
 -   MongoDB v3.x server running
+-   AMQP 0-9-1 server with `foo/bar` vHost created (lile RabbitMQ v3 server running)
+    -   You can set up Mosquitto to run the test as follows:
```

Comment:
You mean mosquitto or rabbitmq? The docker command uses `rabbitmq:3` ...

Reply:
:sweat_smile: I mean RabbitMQ, I don't know why I tend to confuse them haha

Reply:
Ok. Thus this suggestion maybe it an acceptable fix:

```suggestion
    -   You can set up RabbitMQ to run the test as follows:
```

Reply:
Fixed in 6b0da23